docs/arch/monitor.md

```diff
@@ -0,0 +1,47 @@
+# Monitor
+
+## Contributors
+
+- Sarah THEOULLE
+- Pauline CONTAT
+- Thomas BROINE
+- Baptiste BRONSIN
+
+## Functionnalities
+
+- Listening events on a Git remote repository
+- Recognition of the event and adapt the action according to the type of the event
+- Calling the controller through an external API
+
+## What
+
+A monitor must listen to some events from a git remote repository and do some actions onto it.
+We need to recognise the type of event, a `PR` (Pull Request) at first.
```

Comment:
You can remove that part. If your initial implementation only supports PRs, this should not be part of your initial spec. Your HTTP API will only allow users to select PRs as an event, and that's how you will document that initial limitation.

Reply:
This section has been revised.